go.mod

```diff
@@ -1,8 +1,10 @@
 module github.com/matttproud/golang_protobuf_extensions
 
-go 1.9
+go 1.16
```

Comment:
I think in the interests of preventing any surprises for users that we'd want to keep the version specifier kept as-is and only move it independently of the retraction rule.

Reply:
I don't think this PR is useful anymore now that v2 is out.

Reply:
On the contrary, the `v1` branch remains useful until people have migrated off of it.  Want to re-open this pull request, and we can take it from there?  It'd be a good due diligence measure.